pkg/sdk/go/policies_test.go

```diff
@@ -75,7 +73,7 @@ func TestCreatePolicy(t *testing.T) {
 			err:   nil,
 		},
 		{
-			desc: addExistingPolicyDesc,
+			desc: "add existing policy",
```

Comment:
Why did we decide to use hard-coded string, rather then var delclaration, as previously?

Reply:
`gocost` suggest declaring a variable as const when you use it more than 5 times. Initially, we were using it more than 5 times but currently, we use it 4 times.

Reply:
While it might sound counter-intuitive, I prefer literal over constant for readability in this context.